packages/core/src/Stores/contract-trade-store.js

```diff
@@ -70,9 +70,17 @@ export default class ContractTradeStore extends BaseStore {
         this.onSwitchAccount(this.accountSwitchListener);
 
         reaction(
-            () => this.last_contract.contract_info,
+            () => this.root_store.portfolio.active_positions,
             () => {
-                if (!isAccumulatorContract(this.last_contract.contract_info?.contract_type)) return;
+                const { symbol } = JSON.parse(
+                    localStorage.getItem('trade_store') ?? sessionStorage.getItem('trade_store') ?? '{}'
```

Comment:
localStorage will be replaced with sessionStorage here https://github.com/binary-com/deriv-app/pull/9831/files